pkg/features/features.go

```diff
@@ -21,6 +21,10 @@ import (
 // 8.11+ - default is enabled
 const defaultTamperProtection = true
 
+// The default value of the disable policy change acks flag if the flag is missing.
+// 9.2 - disabled (acks are sent)
+const defaultDisablePolicyChangeAcks = false
```

Comment:
I've added this as a feature toggle, but we will need to expose the setting through the UI before we can disable acks from the agent.

The fleet-server will do a little extra work on checkins (comparing API key hashes), but this will not result in extra requests being made to ES.